tools/hack/golang/hack/lib/OWNERS

```diff
@@ -0,0 +1,2 @@
+reviewers:
```

Comment:
omit the file if you don't want to have me in there

Reply:
Hrm, I think i need to take OWNERS out of this dir - we should be at the top level.